docs/guides/1.20.0-upgrade-guide.md

```diff
@@ -43,9 +43,6 @@ The Terraform MongoDB Atlas Provider version 1.20.0 has a number of new and exci
   Note: Applying the non-empty plan will be a no-op but users will still get the non-empty plan even after applying unless one of the above solutions are adopted.
 
 
-## New Terraform MongoDB Atlas modules
```

Comment:
my suggestion is to keep it so it's not only in 1.19 guide and more customers can learn about this.  Maybe we can delete the "New" word and keep the rest